requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ aiofiles==23.2.1
     # via quart
 airspeed-ext==0.6.5
     # via localstack-core
-amazon-kclpy==2.1.3
+amazon-kclpy==2.1.5
```

Comment:
This upgrade already contains `amazon-kclpy==2.1.5` which verifies that this release fixes https://github.com/awslabs/amazon-kinesis-client-python/issues/247.